community/btpd/build

```diff
@@ -2,7 +2,7 @@
 
 autoreconf --verbose --install --force
 
-./configure
+./configure --prefix=/usr
```

Comment:
```sh
./configure \
    --prefix=/usr
```

I suggest reading: https://k1ss.org/style

Reply:
The rush.